lib/tasks/position_openings.rake

```diff
@@ -68,7 +68,7 @@ namespace :jobs do
       puts "Example YAML file syntax:"
       puts "bloomingtonmn:"
       puts "\ttags: city tag_2"
-      puts "\torganization_id: US-MN:CITY-BLOOMINGTON"
+e     puts "\torganization_id: US-MN:CITY-BLOOMINGTON"
```

Comment:
hmmm...not seeing a typo.  looks like it was just an edit formatting thing that affected that line.  Am i missing something?